static/app/appQueryClient.tsx

```diff
@@ -0,0 +1,73 @@
+import {createAsyncStoragePersister} from '@tanstack/query-async-storage-persister';
+import {persistQueryClient} from '@tanstack/react-query-persist-client';
+import {del as removeItem, get as getItem, set as setItem} from 'idb-keyval';
+
+import {SENTRY_RELEASE_VERSION} from 'sentry/constants';
+import {DEFAULT_QUERY_CLIENT_CONFIG, QueryClient} from 'sentry/utils/queryClient';
+
+/**
+ * Named it appQueryClient because we already have a queryClient in sentry/utils/queryClient
```

Comment:
Can we have this comment explain the difference to try and help avoid people using this one if they don't need it?